documentation/getting_started/mqtt-client-openshift.adoc

```diff
@@ -166,17 +166,11 @@ client.connect(opts.connectHost, opts.portHost, 60)
 client.loop_forever()
 ----
 
-
-[options="nowrap",subs=attributes+]
-....
-./tls_mqtt_recv.py -c "$({OcGetRoute} mqtt)" -p 443 -t mytopic -q 1 -s ./server-cert.pem
-....
-
```

Comment:
@lulf I got the reason for not using the oc extract so I was missing the reason. Thanks ! But what about this receiver line ... is it just a mistake for being here ?

Reply:
@lulf Can I close this PR and get rid of this lines in the master ?